ietf-layer0-types-ext.yang

```diff
@@ -649,8 +681,14 @@ module ietf-layer0-types-ext {
       type frequency-ghz;
       config false;
       description
-        "Carrier bandwidth occupancy";
+        "Carrier bandwidth occupancy.
+         The required bandwidth can be given 
+         by the transceiver vendor or
+         can be function of the following parameters:
+         baudrate, nyquist-spacing, roll-off and 
+         cross-talk penalty";

```

Comment:
Should we name the YANG attributes (e.g., available-baud-rate, nyquist-spacing-factor and xtalk-penalty)?